### PR TITLE
Make it possible to disable the dashboards

### PR DESCRIPTION
**Pull request by Petervw**

### CODE REVIEW

File: includes/class-options.php [hunk 1]
```diff
@@ -158,6 +157,23 @@ public function get_tracking_code() {
 		}
 
 		/**
+		 * Convert a checkbox value to a bool
+		 *
+		 * @param $option_name
+		 *
+		 * @return bool
+		 */
+		public function checkbox_value_to_bool( $option_name ) {
```
Comment (andizer) [inline includes/class-options.php]: I should give this a general name like: ```option_value_to_bool```

Comment (Petervw) [inline includes/class-options.php]: Fixed!

File: admin/dashboards/class-admin-dashboards.php [hunk 2]
```diff
@@ -54,58 +63,18 @@ protected function __construct() {
 		 * @param integer $ga_profile_id
 		 */
 		public function init_dashboards( $ga_profile_id ) {
+			$dashboards = $this->get_default_dashboards();
```
Comment (andizer) [inline admin/dashboards/class-admin-dashboards.php]: You can move this into the if-statement. If dashboards are disabled there isn't a need to fill the variable ```$dashboards```

File: includes/class-settings.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	 * @var
+	 */
+	private $options;
+
+	/**
+	 * Set the options of Google Analytics
+	 */
+	protected function __construct() {
+		$this->options_class = Yoast_GA_Options::instance();
+		$this->options       = $this->options_class->get_options();
```
Comment (andizer) [inline includes/class-settings.php]: For now this property isn't used in the class. Is it necessary to init this one?

Comment (Petervw) [inline includes/class-settings.php]: We need this in the future, like the Tracking code function from the options-class. Can we leave this functionality for now, or should we remove it? @omarreiss what to do you think?

**Pull request merged**